rj_gameplay/stp/utils/constants.py

```diff
@@ -1,6 +1,7 @@
 # import robocup
 import stp.rc as rc
 import math
+import sys
 
 # for degrees->rad or reverse, use np.deg2rad or np.rad2deg)
 OUR_CHIP_ROLL = 3  # Average distance at which a ball is slow enough to capture
```

Comment:
Can you delete OUR_CHIP_ROLL and THEIR_CHIPPING? I have no idea where the constants came from and they appear unused elsewhere.

Reply:
(Let me know if I'm wrong and deleting them breaks something)